### PR TITLE
[Gene] Fix pagination.

### PR DESCRIPTION
**Pull request by alloy**
Fixes https://github.com/artsy/collector-experience/issues/907

Also moves to a better architected version of filtered artworks for a gene, which Reaction uses atm as well.

### CODE REVIEW

File: src/lib/Components/ArtworkGrids/RelayConnections/GeneArtworksGrid.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { createPaginationContainer, graphql } from "react-relay"
+import { ConnectionData, createPaginationContainer, graphql } from "react-relay"
 import InfiniteScrollArtworksGrid from "../InfiniteScrollGrid"
 
 import Artwork from "../Artwork"
```
Comment (l2succes) [inline src/lib/Components/ArtworkGrids/RelayConnections/GeneArtworksGrid.tsx]: somewhat unrelated to this PR. Is it still required to for Relay to work properly? (The unused import)

File: src/lib/relay/config.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ if (Emission && Emission.gravityAPIHost && Emission.metaphysicsAPIHost) {
   metaphysicsURL = "https://metaphysics-production.artsy.net"
   gravityURL = "https://api.artsy.net"
 }
+metaphysicsURL = "http://localhost:5001"
```
Comment (l2succes) [inline src/lib/relay/config.tsx]: Remove this before merging

Comment (alloy) [reply src/lib/relay/config.tsx]: Oooohhhhh, good catch!

File: src/lib/Components/ArtworkGrids/RelayConnections/GeneArtworksGrid.tsx (hunk 1 above)
Comment (alloy) [reply src/lib/Components/ArtworkGrids/RelayConnections/GeneArtworksGrid.tsx]: Oh yeah, I don’t actually think so, good point, I will file a ticket for it to follow-up in 4.0.1

File: src/lib/relay/config.tsx (hunk 2 above)
Comment (alloy) [reply src/lib/relay/config.tsx]: Thanks for catching that! 🙏

Comment (l2succes) [reply src/lib/relay/config.tsx]: De nada, I do it all the time 🙂

File: src/lib/Components/ArtworkGrids/RelayConnections/GeneArtworksGrid.tsx (hunk 1 above)
Comment (alloy) [reply src/lib/Components/ArtworkGrids/RelayConnections/GeneArtworksGrid.tsx]: https://github.com/artsy/emission/issues/946

File: src/lib/Containers/__tests__/__snapshots__/Gene-tests.tsx.snap [hunk 3]
```diff
@@ -34,38 +34,19 @@ exports[`looks like expected 1`] = `
     }
   >
     <ArtworksGrid
-      gene={
+      filtered_artworks={
         Object {
-          "_id": "a UUID",
```
Comment (orta) [inline src/lib/Containers/__tests__/__snapshots__/Gene-tests.tsx.snap]: should this be worrying? The artwork metadata was remove

Comment (alloy) [reply src/lib/Containers/__tests__/__snapshots__/Gene-tests.tsx.snap]: This is what it looks like:

simulator screen shot 12 jan 2018 16 45 20

**Comment by alloy** [comment]
Alright, going for the merge.

**Pull request merged**